app/lib/actions/katello/host/update_release_version.rb

```diff
@@ -0,0 +1,29 @@
+module Actions
+  module Katello
+    module Host
+      class UpdateReleaseVersion < Actions::EntryAction
+        def plan(host, release_version)
+          if host.content_facet && host.subscription_facet
+            if release_version.present? && !host.content_facet.available_releases.include?(release_version)
+              fail _("Host %s cannot be assigned release version %s.") % [host.name, release_version]
+            else
+              host.subscription_facet.release_version = release_version
+            end
+            plan_action(Actions::Katello::Host::Update, host)
+            plan_self(:hostname => host.name)
```

Comment:
Currently, if an error occurs the task list will just show "update release version for host".  But if you move this line first in the method , it will show the full hostname in the list